### PR TITLE
Cleanup proxy code

### PR DESCRIPTION
**Pull request by rob-c**
Just a minor code cleanup addressing #566 

Fixing #566

### CODE REVIEW

File: python/GangaLHCb/ganga-proxy.py [hunk 1]
```diff
@@ -28,31 +28,11 @@
...
-                    final_str += str(os.path.expandvars(line)).strip()
-                    if not final_str.endswith(';'):
-                        final_str += " ;"
-                    final_str += " "
-                global_env[k] = final_str
-
-    return global_env
-
-if "GANGADIRACENVIRONMENT" not in get_env():
+temp_env = os.environ()
```
Comment (drmarkwslater) [inline python/GangaLHCb/ganga-proxy.py]: Isn't `os.environ` just a dictionary? I don't think the brackets are needed and the `temp_env` overall seems a bit excessive as you can just use `os.environ` instead.

Comment (rob-c) [inline python/GangaLHCb/ganga-proxy.py]: Fixed :)

**Comment by drmarkwslater** [comment]
Assuming all the other env stuff is correct and this isn't needed anymore then 👍

**Comment by milliams** [comment]
This looks good to me :+1:

**Pull request merged**